pimd/pim_cmd.c

```diff
@@ -8174,11 +8174,12 @@ DEFUN (interface_ip_igmp_query_interval,
 
 DEFUN (interface_no_ip_igmp_query_interval,
        interface_no_ip_igmp_query_interval_cmd,
-       "no ip igmp query-interval",
+       "no ip igmp query-interval [(1-65535)]",
        NO_STR
        IP_STR
        IFACE_IGMP_STR
-       IFACE_IGMP_QUERY_INTERVAL_STR)
+       IFACE_IGMP_QUERY_INTERVAL_STR
+       IGNORED_IN_NO_STR)
 {
 	char default_query_interval[5];
 
```

Comment:
Same here: either stringnify define or use `DESTROY` in the YANG op.

Reply:
chnged to destroy